tests/resources/mock.template.json

```diff
@@ -186,6 +186,15 @@
         "environmentVariable": "apimEndpointSubnetCount"
       }
     },
+    "sharedWorkerSubnetCount": {
+      "type": "int",
+      "defaultValue": 0,
```

Comment:
Default value here is also different to environment.template.json

Reply:
Set to 1 along with other subnet count parameters